django/core/management/commands/startapp.py

```diff
@@ -12,3 +15,11 @@ def handle(self, **options):
         app_name = options.pop("name")
         target = options.pop("directory")
         super().handle("app", app_name, target, **options)
+
+        if target is None:
+            target = os.getcwd()
+        else:
+            target = os.path.abspath(target)
+
+        sys.stdout.reconfigure(encoding="utf-8")
```

Comment:
I don’t understand why we have this call here but not for `startproject`? We also don’t seem to do this for any other CLI output in Django so I’m not sure why this would be needed, and if it is, could warrant a comment to explain the "why".

Reply:
`sys.stdout.reconfigure` was called in order to avoid CI failures. However I attempted to remove it and CI still fails. So maybe I can add a comment, something as 'called in order to prevent encoding error on windows'. What are your thoughts?